02-first_steps.Rmd

```diff
@@ -240,6 +240,177 @@ adsl %>%
   kable_classic_2(full_width = F)
 ```
 
+### Handling of missing values
+
+Missing values in SAS and R:
+
+* SAS 
+  + Missing value is a blank/a single decimal point for character/numeric variables, see details in [Missing Values in SAS](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.5/lrcon/p175x77t7k6kggn1io94yedqagl3.htm);
+  
+* R
+  + Missing data in R appears as `NA`. `NA` is not a string nor a numeric value, but an indicator of missingness;
+  + `NA` and "" are different in R: "" is a blank string, while `NA` is missing;
+  + "Missing values (`NA`) and `NaN` values are regarded as non-comparable even to themselves, so comparisons involving them will always result in `NA`" 
+  + `admiral::convert_blanks_to_na()` can turn SAS blank strings into proper R `NA` (from admiral documentation).  
```

Comment:
I would add a link to the documentation of `convert_blanks_to_na()`: https://pharmaverse.github.io/admiral/reference/convert_blanks_to_na.html